.github/workflows/build-release.yml

```
@@ -35,14 +35,27 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install wheel setuptools
+        pip install wheel setuptools rjsmin rcssmin
     
     - name: Build tar.gz
       run: python3 setup.py sdist
     
     - name: Build wheel
-      run: python3 setup.py bdist_wheel
-    
+      run: |
+        bundlejs="$(sha224sum assets/scripts/*.js | sha224sum | cut -d ' ' -f1).js"
+        bundlecss="$(sha224sum assets/styles/*.css | sha224sum | cut -d ' ' -f1).css"
+        sed '/<script.*src=/d' bin/views/*.html
+        sed "/<link.*rel=[\"']?stylesheet/d" bin/views/*.html
+        echo "<script type=javascript src=\"/assets/${bundlejs}\" defer></script>" >> bin/views/head.html
+        echo "<link rel=stylesheet href=\"/assets/${bundlecss}\">" >> bin/views/head.html
+        cat bin/assets/scripts/*.js | python -m rjsmin > ${bundlejs}
+        cat bin/assets/styles/*.css | python -m rcssmin > ${bundlecss}
+        rm bin/assets/scripts/*.js
+        rm bin/assets/styles/*.css
+        mv ${bundlejs} bin/assets/
+        mv ${bundlecss} bin/assets/
+        python3 setup.py bdist_wheel
```

```suggestion
        gzip bin/assets/${bundlejs}
        gzip bin/assets/${bundlecss}
        python3 setup.py bdist_wheel
```
and enable `gzip_static` in nginx https://stackoverflow.com/a/6952804/4241798